### PR TITLE
Using DataTorrent rss feed for test

### PR DESCRIPTION
**Pull request by pramodin**

### CODE REVIEW

**Comment by pramodin** [comment]
@tweise please see

File: contrib/src/test/java/com/datatorrent/contrib/romesyndication/RomeSyndicationOperatorTest.java [hunk 1]
```diff
@@ -18,25 +18,18 @@
...
 
-import com.datatorrent.contrib.romesyndication.RomeFeedEntry;
-import com.datatorrent.contrib.romesyndication.RomeStreamProvider;
-import com.datatorrent.contrib.romesyndication.RomeSyndicationOperator;
-import com.datatorrent.common.util.BaseOperator;
 import com.datatorrent.api.DAG;
 import com.datatorrent.api.DefaultInputPort;
 import com.datatorrent.api.LocalMode;
+import com.datatorrent.common.util.BaseOperator;
+import org.junit.*;
```
Comment (243826) [inline contrib/src/test/java/com/datatorrent/contrib/romesyndication/RomeSyndicationOperatorTest.java]: whats the rule again? \* or no *?

Comment (davidyan74) [inline contrib/src/test/java/com/datatorrent/contrib/romesyndication/RomeSyndicationOperatorTest.java]: I remember it's no *

Comment (pramodin) [inline contrib/src/test/java/com/datatorrent/contrib/romesyndication/RomeSyndicationOperatorTest.java]: Yes wonder why the idea settings didn't take..

**Comment by tweise** [comment]
Please refer to MLHR-1899 in commit message.

**Comment by pramodin** [comment]
Addressed both your comments, thanks @243826, @tweise

**Pull request merged**